pagerduty/resource_pagerduty_automation_actions_action_test.go

```diff
@@ -84,6 +85,7 @@ func TestAccPagerDutyAutomationActionsActionTypeProcessAutomation_Basic(t *testi
 					resource.TestCheckResourceAttrSet("pagerduty_automation_actions_action.foo", "modify_time"),
 					resource.TestCheckResourceAttrSet("pagerduty_automation_actions_action.foo", "runner_id"),
 					resource.TestCheckResourceAttr("pagerduty_automation_actions_action.foo", "runner_type", "runbook"),
+					resource.TestCheckResourceAttr("pagerduty_automation_actions_action.foo", "only_invocable_on_unresolved_incidents", "true"),
```

Comment:
please, test the "false" value here setting explicitly the value in `testAccCheckPagerDutyAutomationActionsActionTypeProcessAutomationConfigUpdated`